sale_payment_method/README.rst

```diff
@@ -0,0 +1,61 @@
+Sale Payment Method
+===================
+
+It adds a payment method on the sales orders and allow to register
+payments entries on sales orders.
+
+This module is low level and works well with the following modules:
+* **Sale Automatic Worflow** and **Sale Payment Method - Automatic Workflow**
+  the payments created with this module will be automatically reconciled
+  with the invoices of the sales orders.
+* Sale Quick Payment: allows to create the payments with a button on the
+  sales orders.
+
+Also, the e-commerce connectors such as the **Magento Connector** or
+**Prestashop Connector** use it to create the external payments.
+
+Installation
+============
+
+Nothing special is required.
+
+Configuration
+=============
+
+The payment methods can be configured in **Sales > Configuration >
+Sales > Payment Methods**.
```

Comment:
Yeah, it works, because lines with same indentation are joined when parsing. You can check it pressing 'View' on top of here :wink: